### DISCUSSION TITLE
primitivesVisible in annotation()

### DISCUSSION THREAD

**Discussion by mpradovelasco** [OpenModelica/OpenModelica#11587] [Q&A] [answered]
Hi, 

I wonder if the primitivesVisible attribute of IconMap and DiagramMap in annotations() is available currently in OM (to do not visible graphical primitives with extends clause). 

Thanks in advance!

**Comment by adeas31** [comment, marked as answer]
It doesn't work as expected at the moment. See #5610

**Comment by mpradovelasco** [comment]
Thank you for your info @adeas31.
